plugin-flex-ts-template-v2/src/feature-library/custom-transfer-directory/config.ts

```diff
@@ -1,7 +1,7 @@
 import { Manager } from '@twilio/flex-ui';
 
 import { getFeatureFlags } from '../../utils/configuration';
-import { ExternalDirectoryEntry } from './types/ServiceConfiguration';
+import { ExternalDirectoryEntry } from './types/DirectoryEntry';
 
 const { enabled = false, use_paste_search_icon = false } = getFeatureFlags()?.features?.custom_transfer_directory || {};
```

Comment:
this should declare the type from the ServiceConfiguration  (i realize this is a hangover from when i wrote it 😊 )

Reply:
Fixed!